Tests/BugsnagConfigurationTests.m

```diff
@@ -113,6 +126,7 @@ - (void)testSetEndpoints {
     XCTAssertEqualObjects([NSURL URLWithString:@"http://sessions.example.com"], config.sessionURL);
 }
 
+#if DEBUG
```

Comment:
Is this only applicable in debug mode? Having trouble seeing why.

Reply:
Yeah - I'll add a bit more context to the block but `NSAssert()`-based behavior is optimized out in release mode and no exception would be thrown.